dependencies/src/test/java/com/google/cloud/tools/opensource/dependencies/RepositoryUtilityTest.java

```diff
@@ -56,7 +55,7 @@ public void testReadBom_coordinates() throws ArtifactDescriptorException {
   public void testReadBom_path() throws Exception {
     Path pomFile = Paths.get("..", "boms", "cloud-oss-bom", "pom.xml");
     ImmutableList<Artifact> artifacts = RepositoryUtility.readBom(pomFile);
-    Truth.assertThat(artifacts).isNotEmpty();
-    Truth.assertThat(artifacts.size()).isEqualTo(167);
+    Assert.assertFalse(artifacts.isEmpty());
```

Comment:
What's wrong with the Truth assertions?

Reply:
They don't add anything to the basic JUnit methods. And sticking to JUnit where possible is simpler.